src/server/config/server-config.ts

```diff
@@ -21,7 +21,7 @@ export const isSmokeTest = process.env.CI_SMOKE_TEST === "true";
 // AWS
 export const AWS_REGION = "eu-west-1";
 export const LOCATION_SERVICE_INDEX_NAME =
-  process.env.LOCATION_SERVICE_INDEX_NAME;
+  process.env.LOCATION_SERVICE_INDEX_NAME ?? "explore.place";
```

Comment:
what's the relevance of explore.place here?

Reply:
This is a good question. I originally put it here because I saw it being used in AWS when I did a location search

<img width="274" alt="Screenshot 2022-08-09 at 14 49 16" src="https://user-images.githubusercontent.com/1377253/183671347-5e526b3b-8080-4861-b96a-f40988e9cc6d.png">

But after doing lots of research it looks like it's just a name you give to the instance of the location SDK you're using. For AWS to track costs or something 🤷 

I added it because for me it kept showing as undefined (maybe something I need to add to my .env file). So looking around in the AWS console it looks like both `undefined` and `explore.place` might have done by me

<img width="914" alt="Screenshot 2022-08-09 at 14 53 11" src="https://user-images.githubusercontent.com/1377253/183671887-1641436b-b673-4707-885c-374c649f5e15.png">

So I updated the default to `fcdo-professional-service-finder` to match the second option in the above image.

Let me know your thoguhts. [Here is the page about place indexes in the asw console](https://eu-west-1.console.aws.amazon.com/location/places/home?region=eu-west-1#/) if you're interested.